### DISCUSSION TITLE
doris对应的数据类型转换

### DISCUSSION THREAD

**Discussion by weakdouqing** [apache/doris#34777] [A - General / Q&A]
Flink Version：1.15.4
Doris Version：2.0.2
flink-doris-connector-1.16 Version：1.5.2
在Flink这边获取到Boolean类型的数据，使用RowData的形式写入Doris，但在Doris中对应的字段类型为Int，这个时候导入数据没有报错，但是在Doris里面查询的时候，该字段为Null
想问一下后续会有这种情况下的对应类型转换嘛？即将true/false转为1/0
或者是直接报错呢？

**Comment by yagagagaga** [comment]
You can add `'sink.properties.strict_mode'='true'` in your doris sink and an Exception occured when cast fail
